test/test_web.rb

```diff
@@ -34,24 +58,37 @@ def perform(a, b)
     end
 
     it 'can show text with any locales' do
-      rackenv = {'HTTP_ACCEPT_LANGUAGE' => 'ru,en'}
-      get '/', {}, rackenv
-      assert_match(/Панель управления/, last_response.body)
-      rackenv = {'HTTP_ACCEPT_LANGUAGE' => 'es,en'}
-      get '/', {}, rackenv
-      assert_match(/Panel de Control/, last_response.body)
-      rackenv = {'HTTP_ACCEPT_LANGUAGE' => 'en-us'}
-      get '/', {}, rackenv
-      assert_match(/Dashboard/, last_response.body)
-      rackenv = {'HTTP_ACCEPT_LANGUAGE' => 'zh-cn'}
-      get '/', {}, rackenv
-      assert_match(/信息板/, last_response.body)
-      rackenv = {'HTTP_ACCEPT_LANGUAGE' => 'zh-tw'}
-      get '/', {}, rackenv
-      assert_match(/資訊主頁/, last_response.body)
-      rackenv = {'HTTP_ACCEPT_LANGUAGE' => 'nb'}
-      get '/', {}, rackenv
-      assert_match(/Oversikt/, last_response.body)
+      page.driver.headers = { 'Accept-Language' => 'ru,en' }
+      visit '/'
+      assert_text('Панель управления')
+      Percy::Capybara.snapshot(page, name: 'Dashboard (Russian)')
```

Comment:
Could you wrap this API in a helper method at the bottom?  Something like `snap(page, name: ...)` would be fine.  I want to make it trivial to remove Percy if necessary.